tools/ci/rules.go

```diff
@@ -0,0 +1,23 @@
+//go:build ignore
+// +build ignore
+
+package gorules
+
+import (
+	"github.com/quasilyte/go-ruleguard/dsl"
+)
+
+func fmtErrorfWithoutArgs(m dsl.Matcher) {
+	m.Match(`fmt.Errorf($msg)`).
+		Report(`fmt.Errorf: change for errors.New($msg)`).
+		Suggest(`errors.New($msg)`)
+}
```

Comment:
I started with this rule just to figure out the DSL, but it turned out to be helpful so I kept it around and fixed the places that used this.